openpype/tools/sceneinventory/switch_dialog.py

```diff
@@ -1290,7 +1294,11 @@ def _trigger_switch(self, loader=None):
                         repre_doc = _repres.get(container_repre_name)
 
             if not repre_doc:
-                version_doc = version_docs_by_parent_id[subset_id]
+                version_by_name = version_docs_by_parent_and_name[subset_id]
+                if selected_asset or selected_subset:
+                    version_doc = last_version_docs_by_parent_id[subset_id]
```

Comment:
Couldn't this just be 
```python
# Use last version
version_name = max(version_by_name)
version_doc = version_by_name[version_name]
```
So that we don't need to separately keep track of the `last_version_docs_by_parent_id`.